### PR TITLE
Move eip-2535 to Review

### PR DESCRIPTION
**Pull request by mudgen**
EIP-2535 has had feedback and been reviewed for many months. It is ready to move to Last Call.

Here is a link to a list of projects that are already using EIP-2535: https://eip2535diamonds.substack.com/p/list-of-projects-using-eip-2535-diamonds

Here is a link to a list of smart contract security audits that have been done that include the EIP-2535 reference implementations: https://eip2535diamonds.substack.com/p/smart-contract-security-audits-for

### CODE REVIEW

**Comment by mudgen** [comment]
@MicahZoltu please help move this EIP to Last Call.

**Review by lightclient — CHANGES_REQUESTED** [review]
Please first move the EIP to review status. Then we can move to last call.

**Comment by mudgen** [comment]
> Please first move the EIP to review status. Then we can move to last call.

Understood. I moved it to review status.

**Comment by mudgen** [comment]
@lightclient Is this okay to move to Review now?

**Review by axic — APPROVED** [review]

**Comment by eip-automerger** [comment]
Hi! I'm a bot, and I wanted to automerge your PR, but couldn't because of the following issue(s):

 - Trying to change EIP 2535 state from Draft to Review

File: EIPS/eip-2535.md [hunk 1]
```diff
@@ -3,7 +3,7 @@ eip: 2535
 title: Diamonds
```
Comment (axic) [inline EIPS/eip-2535.md:3]: As a review comment: I do not think this is a good title. I suggest to improve this before going for Last Call.

Comment (mudgen) [reply EIPS/eip-2535.md:3]: Understood and appreciated.  Any ideas on a better title?

Comment (mudgen) [reply EIPS/eip-2535.md:3]: @axic How about "Diamonds, Multi-Faceted Proxy Contracts"   ?

Comment (mudgen) [reply EIPS/eip-2535.md:3]: I created a poll about the name of the standard: https://twitter.com/mudgen/status/1419840710963081222

Comment (mudgen) [reply EIPS/eip-2535.md:3]: I created a second poll for the name of the standard here: https://twitter.com/mudgen/status/1419865781169446915

**Pull request merged**